src/c_kzg_4844.c

```diff
@@ -1253,66 +1258,86 @@ C_KZG_RET compute_blob_kzg_proof(
 }
 
 /**
- * Given a blob and its proof, verify that it corresponds to the provided
- * commitment.
- *
- * @param[out] ok               True if the proofs are valid, otherwise false
+ * Given a blob and its proof and commitment as bytes, compute a
+ * kzg_evaluation_claim by parsing these into group elements and computing the
+ * Fiat-Shamir challenge for verification. The resulting claim consists of a
+ * commitment to a polynomial p, an evaluation point, a puported evaluation
+ * result p(evaluation_point) and a purported proof for that.
+ *
+ * @param[out] claim            Corresponding claim & proof about a polynomial
+ * identity
  * @param[in]  blob             Blob to verify
  * @param[in]  commitment_bytes Commitment to verify
  * @param[in]  proof_bytes      Proof used for verification
  * @param[in]  s                The trusted setup
  */
-C_KZG_RET verify_blob_kzg_proof(
-    bool *ok,
+C_KZG_RET make_claim_from_blob_kzg_proof(
```

Comment:
1. You don't need to tell what you make the claim from. People can tell from the parameter names
2. Reusing the verb from other functions also help make the code base conformant

So I think naming this function `compute_claim` makes more sense.